src/overlays/actors/ovl_Shot_Sun/z_shot_sun.h

```diff
@@ -3,15 +3,20 @@
 
 #include "global.h"
 
+#define SHOTSUN_GET_FF(thisx) ((thisx)->params & 0xFF)
+
 struct ShotSun;
 
 typedef void (*ShotSunActionFunc)(struct ShotSun*, PlayState*);
 
 typedef struct ShotSun {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x4C];
-    /* 0x0190 */ ShotSunActionFunc actionFunc;
-    /* 0x0194 */ char unk_194[0xC];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ ColliderCylinder collider;
+    /* 0x190 */ ShotSunActionFunc actionFunc;
+    /* 0x194 */ Vec3s hitboxPos;
+    /* 0x19A */ s16 timer;
+    /* 0x19C */ s8 unused;
+    /* 0x19D */ UNK_TYPE1 pad19D[3];
 } ShotSun; // size = 0x1A0
```

Comment:
This pad seems to be just struct padding, you can remove it.
```suggestion
    /* 0x19C */ s8 unused;
} ShotSun; // size = 0x1A0
```